addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7614,7 +7614,16 @@ msgctxt "#13457"
 msgid "Prefer VAAPI render method"
 msgstr ""
 
-#empty strings from id 13458 to 13459
+#: system/settings/settings.xml
+msgctxt "#13458"
+msgid "Use Decoder filter"
+msgstr ""
```

Comment:
Could you please add ''#.' line that verbally describes what this setting is exactly doing. 

I think the translators will really need this to find the correct translation. Most of them are no tech nerds so might have no idea what a decoder filter is

Reply:
Sure, I'll add a description about it later